source/extensions/filters/http/lua/config.cc

```diff
@@ -35,8 +35,8 @@ LuaFilterConfig::createRouteSpecificFilterConfigTyped(
 /**
  * Static registration for the Lua filter. @see RegisterFactory.
  */
-LEGACY_REGISTER_FACTORY(LuaFilterConfig, Server::Configuration::NamedHttpFilterConfigFactory,
-                        "envoy.lua");
+REGISTER_FACTORY(LuaFilterConfig, Server::Configuration::NamedHttpFilterConfigFactory);
+REGISTER_FACTORY(UpstreamLuaFilterConfig, Server::Configuration::UpstreamHttpFilterConfigFactory);
```

Comment:
Could we keep the legacy filter name `envoy.lua`, at least for the downstream lua. Thanks.

Reply:
donez.